Zonk_2.0/main.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <malloc.h>
+#include <windows.h>
+#include <stdbool.h>
+#include "GAME_LOGIC.h"
+#include "Utils.h"
+
+
+int main()
+{
+    int usedDice = 0, sum = 0, round = 1;
+    int* dices = NULL, sortedDice[6];
+    int playerResult = 0, playerDecision = 1, selectedDice[6];
+    int botResult = 0, botDecision = 1;
+
+    srand(time(NULL));
+
+    printf("\t\t\t\t\t\tNew game !\n\n");
+
+    while (!isWon(playerResult) && !isWon(botResult))
```

Comment:
There is no need in this first loop. You may check whether player is won or not (similar with the bot) after the loop on 27 line.